spec/jobs/job_configurations_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'job_configurations' do
```

Comment:
We also have a corresponding test that matches the exact file path, can we move these specs there? `spec/config/initializers/job_configurations_spec.rb`

Reply:
I'm surprised I missed that